widget/embedded/src/store/slices/data.ts

```diff
@@ -117,6 +139,9 @@ export const createDataSlice: StateCreator<
       cacheService.set(cacheKey, supportedTokens);
     }
 
+    supportedTokens = supportedTokens.concat(customTokens);
```

Comment:
Shouldn't we check for customToken in config to see if we should concat customTokens?
Also please remove the console log.

Reply:
fixed